packages/buckets/src/buckets.ts

```diff
@@ -617,6 +619,9 @@ export class Buckets extends GrpcAuthentication {
    * ```
    */
   async pushPath(key: string, path: string, input: any, options?: PushOptions): Promise<PushPathResult> {
+    if (isNode) {
+      return bucketsPushPathNode(this, key, path, input, options)
```

Comment:
maybe it should just run this method every time.

Reply:
as in, even when not isNode

Reply:
Yeh, I don't think there's anything actually node specific happening here, so we could just run this way every time.